data/migrations/deb/1_9_1_to_1_9_2.py

```diff
@@ -0,0 +1,79 @@
+import os
+import shutil
+import subprocess
+import sys
+
+from indy_common.config_helper import NodeConfigHelper
+from indy_common.config_util import getConfig
+from stp_core.common.log import getlogger
+
+
+logger = getlogger()
+
+ENV_FILE_PATH = "/etc/indy/indy.env"
+BUILDER_NET_NETWORK_NAME = "net3"
+
+
+def get_node_name():
+    node_name = None
+    node_name_key = 'NODE_NAME'
+
+    if os.path.exists(ENV_FILE_PATH):
+        with open(ENV_FILE_PATH, "r") as fenv:
+            for line in fenv.readlines():
+                if line.find(node_name_key) != -1:
+                    node_name = line.split('=')[1].strip()
+                    break
+    if node_name is None:
+        logger.error("{} file doesn't contains a node name. Please add string like NODE_NAME=<node name> into this file".format(ENV_FILE_PATH))
+
+    return node_name
+
+
+def is_node_stopped():
+    ret = subprocess.run(["systemctl", "status", "indy-node"], stdout=subprocess.PIPE)
+
+    if ret.returncode == 0:
+        return False
+    return True
+
+
+def migrate_all():
+
+    # Check, that node is stopped
```

Comment:
Not sure that we need these steps, since this is part of migration logic, and the node will be stopped for sure.